magento/pwa-studio
# [Doc] Update README

## Description

Updates the project README file with more up to date information.

This PR also updates the dependencies for the pwa-devdocs project

## Related Issue





Closes PWA-883

## Acceptance 

@ericerway 
Any developer

### Verification Stakeholders

Any developer

### Specification



### Verification Steps

1. View the README.md file
1. Verify changed content

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
